example/lib/main.dart

```diff
@@ -112,16 +131,16 @@ class MyHomePage extends StatelessWidget {
               ElevatedButton(
                 child: const Text('Provide feedback'),
                 onPressed: () {
-                  BetterFeedback.of(context)?.show(
+                  BetterFeedback.of(context)!.show(
```

Comment:
`!` is preferable as it'll fail loudly instead of silently.

Reply:
Yeah, that's definitly something I want to tackle, along with making the image not nullable.
In the past I wasn't comfortable making it non nullable but now it's quite popular and nobody complained that it fails.